packages/server/src/documents/documents.controller.ts

```diff
@@ -111,28 +148,7 @@ export class DocumentsController {
         },
       },
     };
-
-    const createResult: Document = await this.centrifugeService.documents.createDocumentV2(
-      request.user.account,
-      {
-        attributes: payload.attributes,
-        read_access: payload.header.read_access
-          ? payload.header.read_access
-          : [],
-        write_access: payload.header.write_access
-          ? payload.header.write_access
-          : [],
-        scheme: CoreapiCreateDocumentRequest.SchemeEnum.Generic,
-      },
-    );
-
-
-    const commitResp = await this.commitPendingDoc(createResult, request);
-    return await this.updateDBDoc(
-      commitResp.commitResult,
-      commitResp.dbId,
-      request.user.account,
-    );
+    return  await this.saveDoc(payload, request.user)
```

Comment:
I added the route as wrappers just in case we wanna do more stuff in the future, add default values or auto commit on save.